app/views/kaminari/_gap.html.erb

```diff
@@ -0,0 +1 @@
+<li class="disabled"><a href="#">...</a></li>
```

Comment:
Did you try to see if it works without copying them? Or alternatively, did you check to see if you change one of these, it is reflected in both wice and non-wice pages? It would be nice to have these ones take precedence...